packages/components/src/presentation-components/instance-filter-builder/Utils.ts

```diff
@@ -184,7 +184,8 @@ function numericPropertyValidator({ property, value, operator }: ValidatorContex
   }
 
   function getErrorMessage() {
-    return property.quantityType === undefined
+    // eslint-disable-next-line @typescript-eslint/no-deprecated
+    return property.kindOfQuantityName === undefined || property.quantityType === undefined
```

Comment:
Looking at this change second time it looks like initial approach was correct. My bad. 😞

```suggestion
    return property.kindOfQuantityName === undefined && property.quantityType === undefined
```